packages/ui-filecoin/src/SimpleCard.tsx

```diff
@@ -36,38 +38,63 @@ export type SimpleCardData = {
   cta: NonNullable<SimpleCardProps['cta']>
 }
 
-type CardLayoutStyles = Record<
-  BorderKey,
-  { inner: string; content: string; cta: string }
->
-
 type CardContentProps = Pick<SimpleCardProps, 'title' | 'description'> & {
-  layout: CardLayoutStyles[BorderKey]
+  border: CardOuterVariants['border']
 }
 
-type BorderKey = keyof typeof borderStyles
+const cardOuter = cva(['group h-full w-full overflow-hidden'], {
+  variants: {
+    border: {
+      all: 'rounded-2xl border border-[var(--color-border-base)]',
+      'only-top': 'border-t border-[var(--color-border-base)]',
+    },
+    interactive: {
+      true: 'focus-within:brand-outline focus-within:bg-[var(--color-card-background-hover)] hover:bg-[var(--color-card-background-hover)] relative',
+      false: null,
+    },
+  },
+  defaultVariants: {
+    border: 'all',
+    interactive: false,
+  },
+})
 
-const borderStyles = {
-  all: 'rounded-2xl border border-[var(--color-border-base)]',
-  'only-top': 'border-t border-[var(--color-border-base)]',
-} as const
+const cardInner = cva('flex flex-col gap-6', {
+  variants: {
+    border: {
+      all: 'p-8',
+      'only-top': 'pt-8 pb-6',
+    },
+  },
+  defaultVariants: {
+    border: 'all',
+  },
+})
 
-const interactiveStyles: Partial<Record<BorderKey, string>> = {
-  all: 'focus-within:brand-outline focus-within:bg-[var(--color-card-background-hover)] hover:bg-[var(--color-card-background-hover)]',
-}
+const cardContent = cva('flex flex-col gap-3', {
+  variants: {
+    border: {
```

Comment:
Why is this called `border`? The styles don't seem to have anything to do with border styles?